.ci/dockerfile.ubuntu16

```diff
@@ -0,0 +1,32 @@
+FROM dealii/ubuntu16
+
+MAINTAINER timo.heister@gmail.com
+
+USER root
+
+# note: remove python after candi works with python3 by default
```

Comment:
That's kind of sad. Is it just because of `print` and `xrange` or there is more?

Reply:
should be an easy fix because we are not really using python, but the script fails right now. Will followup later.